daemon/daemon.go

```diff
@@ -181,21 +181,38 @@ func (d *Daemon) Init() error {
 		return err
 	}
 
-	if len(listeners) != 2 {
-		return fmt.Errorf("daemon does not handle %d listeners right now, only two", len(listeners))
+	listenersCount := len(listeners)
+	if listenersCount > 2 {
```

Comment:
As mentioned earlier in the channel, let's please drop this. This breaks forward compatibility (future daemon supports more).